model/src/main/java/com/ge/predix/acs/rest/PolicyEvaluationRequestV1.java

```diff
@@ -119,9 +119,7 @@ public boolean equals(final Object obj) {
                 return false;
             }
             if ((null != this.subjectAttributes) && (this.subjectAttributes.size() == other.subjectAttributes.size())) {
-                for (int i = 0; i < this.subjectAttributes.size(); i++) {
-                    equalsBuilder.append(this.subjectAttributes.get(i), other.subjectAttributes.get(i));
-                }
+                equalsBuilder.append(this.subjectAttributes, other.subjectAttributes);
```

Comment:
I don't believe this is the equivalent of comparing all items for equality. HashSet.equals() does not call equals() for every item in the set. See See https://docs.oracle.com/javase/7/docs/api/java/util/Set.html